src/commcare_cloud/ansible/roles/elasticsearch/templates/config/elasticsearch.yml.j2

```diff
@@ -81,20 +97,56 @@ cloud:
 {% endif %}
 
 # Only allow indices to be created by an explicit "create" command
+{% if elasticsearch_version == '2.4.6'%}
 action.auto_create_index: false
+{% endif %}
+{% if elasticsearch_version is version('7.0.0', '>=') %}
+action.auto_create_index: .watches,.triggered_watches,.watcher-history-*
+{% endif %}
 
 
 {% if es_local_repo and not backup_es_s3%}
 path.repo: ["{{ elasticsearch_data_dir }}/snapshot"]
 {% endif %}
 
 
+
+{% if elasticsearch_version is version('7.0.0', '>=') %}
+cluster.initial_master_nodes:
+{% for item in groups.elasticsearch %}
+{% if hostvars[item].elasticsearch_master is defined|default(False) %}
+{% if item|ipaddr %}
+    - "{{ item }}"
+{% else %}
+    - "{{ lookup('dig', item, wantlist=True)[0] }}"
+{% endif %}
+{% endif %}
+{% endfor %}
+{% endif %}
+
+
+{% if elasticsearch_version == '2.4.6'%}
 # see also discovery.zen.minimum_master_nodes above for how a master gets elected
 {# only include master eligibility if it is set on at least one node in the cluster #}
 {% if groups.elasticsearch|map('extract', hostvars, 'elasticsearch_master')|select()|list %}
-node.master: {{ elasticsearch_master|default('false') }}
+node.master: {{ elasticsearch_master|default(False) }}
```

Comment:
I'm not sure if this needs to be `true/false` or if `True/False` is also OK. The ES docs use lowercase which I think is why this was a string before.

Reply:
Ah, right. Updated to what it was